### ISSUE TITLE
maltparser log4j dependency

### ISSUE THREAD

**Issue by engineerchange** [nltk/nltk#2915]
I wanted to note that malt.py appears to have a log4j dependency. Any recommendations on how to eliminate any concerns with the recent exploit?

**Comment by tomaarsen** [comment]
MaltParser is not packaged within NLTK, thus general use of NLTK should not be vulnerable to log4j.

To use NLTK's `malt.py`, then you must also install a version of a [MaltParser jar](http://maltparser.org/). Then, NLTK simply calls the MaltParser jar. If you wish to avoid a potential Log4j vulnerability, then I would advise downloading a new MaltParser jar to some version without this dependency/vulnerability, if one exists. If one does not exist, then I would recommend to raise your concerns to the MaltParser developers instead, or create a community version that has the exploit patched. 

Long story short - you may only be vulnerable if:
- you have installed the [MaltParser jar](http://maltparser.org/),
- you are using the [`MaltParser`](https://github.com/nltk/nltk/blob/develop/nltk/parse/malt.py#L26) parser imported from `nltk`, and
- you are a version of the [MaltParser jar](http://maltparser.org/) that has not been patched. 

I would not be concerned unless all 3 of these are true for you.

- Tom Aarsen
